platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/src/test/java/com/swirlds/platform/test/sync/SyncTests.java

```diff
@@ -687,13 +688,15 @@ void testSendExpiredEvents() throws Exception {
     @ParameterizedTest
     @MethodSource({"tenNodeGraphParams", "tenNodeBigGraphParams", "tipExpiresBreakingSeed"})
     void tipExpiresAfterPhase1(final SyncTestParams params) throws Exception {
-        final long creatorIdToExpire = 0;
         final SyncTestExecutor executor = new SyncTestExecutor(params);
         final AtomicLong maxGen = new AtomicLong(EventConstants.GENERATION_UNDEFINED);
 
+        final NodeId creatorIdToExpire = executor.getAddressBook().getNodeId(0);
+        final int creatorIndex = executor.getAddressBook().getIndexOfNodeId(creatorIdToExpire);
```

Comment:
`creatorIndex` is just 0, right?

Reply:
lol.  yeah I guess it is.

Reply:
The `creatorIdToExpire` is used elsewhere.   I swapped the order of initializing the variables.